lib/DDG/Spice/PoliticalPolls.pm

```diff
@@ -0,0 +1,42 @@
+package DDG::Spice::PoliticalPolls;
+# ABSTRACT: Election and political polls from HuffPost Pollster
+
+use DDG::Spice;
+
+# Caching - https://duck.co/duckduckhack/spice_advanced_backend#caching-api-responses
+spice is_cached => 1;
+
+# Metadata.  See https://duck.co/duckduckhack/metadata for help in filling out this section.
+name "Political Polls";
+source "HuffPost Pollster";
+icon_url "http://elections.huffingtonpost.com/favicon.ico";
+description "Provides polling data from the United States.";
+primary_example_queries "US polls", "polls";
+secondary_example_queries "election polls";
+# Uncomment and complete: https://duck.co/duckduckhack/metadata#category
+category "reference";
+# Uncomment and complete: https://duck.co/duckduckhack/metadata#topics
+topics "special_interest";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/PoliticalPolls.pm";
+attribution github => ["alexander95015", "Alexander"],
+            facebook => "alexander95015",
+            twitter => "alexander95015";
+
+spice wrap_jsonp_callback => 1;
```

Comment:
It looks like it can accept a callback parameter so I think we can get rid of this? http://elections.huffingtonpost.com/pollster/api